src/ClinicalDecisionRules/Interface/RuleLibrary/RuleManager.php

```diff
@@ -331,23 +331,6 @@ function getRuleFilterCriteria($rule, $guid)
         return null;
     }
 
-    /**
-     * @param string $guid
-     * @return array of OpenEMR\ClinicalDecisionRules\Interface\RuleLibrary\RuleTargetActionGroup
-     */
-    function getRuleTargetActionGroups($rule)
```

Comment:
There are no cases in the openemr codebase where this function is used, and if it is used somewhere else, it is either always crashing or `sizeof($criterion)` is always 0. In any case, this function is broken in more than one way and does not make sense.